assets/setup-wizard/features/installation-feedback.js

```diff
@@ -107,41 +104,7 @@ const InstallationFeedback = ( { onContinue, onRetry } ) => {
 
 	return (
 		<div className="sensei-setup-wizard__features-installation-feedback">
-			<List
```

Comment:
For now, I just removed this part to remove the component that we won't use anymore based on the new design/behavior. But the whole Features step will be removed later or changed to some of the other steps.